packages/react-celo/src/connectors/wallet-connect.ts

```diff
@@ -181,31 +219,6 @@ export default class WalletConnectConnector
     }
   };
 
-  private combinedSessionUpdater(session: WCSession) {
```

Comment:
this was specific to the way wcv1 handled chain and address changes and as such is no longer needed